utils/flare.py

```diff
@@ -74,7 +74,7 @@ class Flare(object):
     ]
     MAIN_CREDENTIALS = [
         CredentialPattern(
-            re.compile('^api_key:( *\w+(\w{5}) ?,?)+$'),
+            re.compile('\s*api_key:( *\w+(\w{5}) ?,?)+$'),
```

Comment:
Could you keep the `^` ? This regex could match other things (commented api_key for instance). I can't think of a case where it really matters, but I'd say let's avoid that altogether and restrict the possible cases.